src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,263 @@
+import classNames from 'classnames';
+import { CSSProperties } from 'vue';
+import { renderHelper } from '../../util/helpers';
+// import CorCheckIcon from '../Icons/CorCheckIcon/CorCheckIcon';
+import Notifier from '../Notifier/Notifier';
+import { NotifierInstance } from '../Notifier/Notifier.types';
+import {
+    ConfigProps,
+    NotificationApi,
+    NotificationArgsProps,
+    NotificationPlacement,
+} from './Notification.types';
+import './Notification.scss';
+import { VueNode } from '../../util/types';
+
+// cached instances
+const notificationInstances: { [key: string]: NotifierInstance } = {};
+
+// default values
+let defaultDuration = 5;
+let defaultTop = '24px';
+let defaultBottom = '24px';
+let defaultPrefixCls = 'cor_notification';
+let defaultPlacement: NotificationPlacement = 'topRight';
+let defaultGetContainer = () => document.body;
+let defaultCloseIcon: VueNode | (() => VueNode) = null;
+let defaultClosable = true;
+let rtl = false;
+let maxCount: number;
+
+// set the config to override defaults
+function setNotificationConfig(options: ConfigProps) {
+    const { duration, placement, bottom, top, getContainer, closeIcon, prefixCls } = options;
+    if (prefixCls !== undefined) {
+        defaultPrefixCls = prefixCls;
+    }
+    if (duration !== undefined) {
+        defaultDuration = duration;
+    }
+    if (placement !== undefined) {
+        defaultPlacement = placement;
+    }
+    if (bottom !== undefined) {
+        defaultBottom = typeof bottom === 'number' ? `${bottom}px` : bottom;
+    }
+    if (top !== undefined) {
+        defaultTop = typeof top === 'number' ? `${top}px` : top;
+    }
+    if (getContainer !== undefined) {
+        defaultGetContainer = getContainer;
+    }
+    if (closeIcon !== undefined) {
+        defaultCloseIcon = closeIcon;
+    }
+    if (options.rtl !== undefined) {
+        rtl = options.rtl;
+    }
+    if (options.maxCount !== undefined) {
+        maxCount = options.maxCount;
+    }
+    if (options.closable !== undefined) {
+        defaultClosable = options.closable;
+    }
+}
```

Comment:
should be an other file

Reply:
if its in another file I can't override default values